packages/pages/src/components/image/image.test.tsx

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+import {Image, getImageUUID, handleLayout} from "./image";
+import {ImageLayout} from "./types";
+import { render, screen } from '@testing-library/react';
+
+const imgWidth = 20;
+const imgHeight = 10;
+const imgUUID = "uuid";
+const width = 200;
+const height = 100;
+const aspectRatio = 1;
+const image = {
+  image: {
+    width: imgWidth,
+    height: imgHeight,
+    url: `https://a.mktgcdn.com/p/${imgUUID}/2x1.jpg`,
+  },
+};
+
+/**
+ * @jest-environment node
+ */
+
+describe("Image", () => {
+  it(`properly logs warning when layout is not ${ImageLayout.FIXED} and width or height is provided`, async () => {
+    jest.clearAllMocks();
+    const logMock = jest.spyOn(console, "warn").mockImplementation(() => {});
+
+    expect(logMock.mock.calls.length).toBe(0);
+    render(<Image
+      image={image}
+      layout={ImageLayout.INTRINSIC}
+      width={100}
+    />);
+    expect(logMock.mock.calls.length).toBe(1);
+
+    jest.clearAllMocks();
+  });
+
+  it("properly renders the image with the pass through style and imgOverrides", () => {
```

Comment:
Is there a way to test the lazy loading-ness of the image?

Reply:
I don't think so. The onLoad on the img tag is not invoked in test so there's no good way to test the loading option